### ISSUE TITLE
The client can't connect to Faraday Server. You can try to reconnect to the last URL you set up, change it or exit Faraday until you fix the problem.  For more information about Faraday Server please refer to the Faraday Github Wiki. 

### ISSUE THREAD

**Issue by jooeji** [infobyte/faraday#188]
***
The client can't connect to Faraday Server. You can try to reconnect to the last URL you set up, change it or exit Faraday until you fix the problem. 
For more information about Faraday Server please refer to the Faraday Github Wiki. 
***
$ git clone https://github.com/infobyte/faraday.git faraday-dev
$ cd faraday-dev
$ ./install.sh
$ ./faraday-server.py
$ ./faraday.py
base in ubuntu 14.04 
i try it first after it installed,but [The client can't connect to Faraday Server. You can try to reconnect to the last URL you set up, change it or exit Faraday until you fix the problem.  For more information about Faraday Server please refer to the Faraday Github Wiki. ]

**Comment by Ezequieltbh** [comment]
Hi @jooeji !
Check ~/.faraday/config/user.xml <couch_uri> path, that should be URL to Faraday server!
And try again!

**Comment by jooeji** [comment]
***
2017-04-08 12:50:01,782 - faraday.launcher - INFO - Starting main application.

*              faraday ui is ready              
Make sure you got couchdb up and running.
If couchdb is up, point your browser to: 
[http://127.0.0.1:5985/_ui]

2017-04-08 12:50:01,785 - faraday - INFO - XMLRPC API server configured on ('localhost', 9876)
2017-04-08 12:50:01,795 - faraday - INFO - REST API server configured on ('localhost', 9977)
2017-04-08 12:50:01,819 - faraday.Server-GTK IO - WARNING - Couldn't get a valid response from the server when requesting to URL http://127.0.0.1:5985/_api/ws and function <function get at 0x7fc00bc271b8>.
***
i used localhost,server and client all in one

**Comment by Ezequieltbh** [comment]
We have a bug and we are reviewing a solution for this!
Sorry for the troubles.
When we have a solution, we can contact you here again!

**Comment by tsluyter** [comment]
I've run into this issue in the past, where Faraday would become unstable upon pausing/resuming the VM that it runs on. Or after sleeping the laptop that the VM runs on. In that case, the workaround is to kill all Faraday processes, restart couchdb and restart the server + IDE.

**Comment by AlexCline** [comment]
I also ran into this issue while setting up a new Faraday v2.4.0 installation on my workstation for testing. I'm not able to use the GUI at all.

I was able to use the WebUI, by creating a new Workspace in a clean installation of v2.4.0 by using the [scripting interface](https://github.com/infobyte/faraday/wiki/Manage-Workspaces#scripting). Then I was able to poke around in the WebUI to perform basic tasks.
